requirements.txt

```diff
@@ -0,0 +1,12 @@
+aiofiles==0.6.0
+aiohttp==3.7.4.post0
```

Comment:
Remove

Reply:
discord.py depends in aiohttp.

Reply:
aiofiles has been removed

Reply:
If it is a requirement of discord.py, then it really does not need to be here. Ideally the only things in this file are the ones that you would need to get up and running. I am assuming this is a `pip freeze`? The problem then is that when we want to change `discord.py` versions, this file will get out of sync.